services/search/pkg/query/kql/cast.go

```diff
@@ -74,5 +76,53 @@ func toTime(in interface{}) (time.Time, error) {
 		return time.Time{}, err
 	}
 
-	return dateparse.ParseLocal(ts)
+	return now.Parse(ts)
+}
+
+func toTimeRange(in interface{}) (*time.Time, *time.Time, error) {
+	var from, to time.Time
+
+	value, err := toString(in)
+	if err != nil {
+		return &from, &to, UnsupportedTimeRangeError{}
+	}
+
+	c := &now.Config{
```

Comment:
@kobergj FYI

Reply:
Hehe. A bit against our new philosophy of having less configuration options :wink: But I'm fine with it for now....

Reply:
same same 🤣 but start of the week needs that...